.storybook/style.css

```diff
@@ -1,4 +1,4 @@
-@tailwind base;
+/* @tailwind base; */
```

Comment:
why do we do this ?

Reply:
<img width="411" alt="image" src="https://github.com/oku-ui/primitives/assets/44074974/9d87cf76-92c7-4ed3-a561-e9b0bc04179a">
<img width="474" alt="image" src="https://github.com/oku-ui/primitives/assets/44074974/dd948fea-cfab-479b-aac8-080b11d9bae3">

html default style can be reseted by taiwind base.

Reply:
So I delete the taiwind base

Reply:
i will take a look at this